game-core/build.gradle

```diff
@@ -18,8 +18,6 @@ plugins {
     id 'de.undercouch.download' version '3.3.0'
 }
 
-apply plugin: 'org.junit.platform.gradle.plugin'
-
```

Comment:
👍
But the dependency is still referenced at the start of the build script.

Reply:
Good catch, fixed in: 25feb59